lib/tasks/heart_rate_data.rake

```diff
@@ -0,0 +1,9 @@
+namespace :heart_rate_data do
+  desc "Update Fitbit heart rate data for the last user"
+  task sync: :environment do
+    user = User.last
+    user.heart_rate_datas << HeartRateData.new
+
+    user.heart_rate_datas.last.update_heart_rate_data!
```

Comment:
Yeah good question. 

Actually, when you run: `user.heart_rate_datas << HeartRateData.new`. Rails has to save the `HeartRateData` object to the DB, in order to create the relationship. So there is actually a newly persisted `HeartRateData` record at this point.

In the future I would like to move this logic into some kind of factory method but I thought it was best to leave the code here for now and wait for future requirements.